Kitodo/src/main/java/org/kitodo/production/forms/ProjectForm.java

```diff
@@ -475,16 +475,20 @@ public Project getProject() {
     }
 
     /**
-     * Set my project.
+     * Set my project and check for assigned processes.
      *
      * @param project
      *            Project object
      */
     public void setProject(Project project) {
-        // has to be called if a page back move was done
         cancel();
         this.project = project;
-        hasProcesses = !project.getProcesses().isEmpty();
+        try {
+            hasProcesses = ServiceManager.getProjectService().hasProcesses(project.getId());
+        } catch (DAOException e) {
+            Helper.setErrorMessage("Error checking for assigned processes", logger, e);
```

Comment:
Sorry forget to add this one: the error message did not get translated right? As this is show to the user a translation would be good or?